pkgs/by-name/re/renpy/package.nix

```diff
@@ -22,8 +22,8 @@ let
   # base_version is of the form major.minor.patch
   # vc_version is of the form YYMMDDCC
   # version corresponds to the tag on GitHub
-  base_version = "8.2.1";
-  vc_version = "24030407";
+  base_version = "8.3.1";
```

Comment:
There is a version mismatch. The latest package version (as it's also mentioned by the PR message / commit message) is 8.3.2 and not 8.3.1.

Reply:
The latest GitHub tag is 8.3.1 though (and 8.3.2 claims to be an Android-specific fix)

Reply:
In this case, should this be named 8.3.1 like the latest git release?

Reply:
I have updated the message to 8.3.1